src/style.css

```diff
@@ -1,3 +1,47 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.logo__img {
+  display: inline-block;
+  position: absolute;
```

Comment:
Using `position: absolute;` for the logo might cause layout issues, especially on smaller screens or when the viewport size changes. Consider using a more flexible layout approach, such as flexbox or grid, to ensure the logo is positioned correctly across different screen sizes.